### PR TITLE
Task#3

### LINKED ISSUE #28: Zadanie 3 (funkcja nextDay)
Dopisz do klasy Ship, Cargo oraz Stock metodę nextDay()

    Klasa Ship: Metoda powinna odejmować po 1 sztuce monety za każdego członka załogi.
    Klasa Cargo: Metoda powinna powodować psucie się towarów.
    Klasa Stock: Metoda powinna zmieniać ilość towaru w sklepach.

### PR DESCRIPTION
**Pull request by Vicx95**

### CODE REVIEW

**Review by arafalski — DISMISSED** [review]

File: shm/inc/item.hpp [hunk 1]
```diff
@@ -22,10 +22,14 @@ class Item : public Cargo {
...
     size_t getBasePrice() const override { return basePrice_; }
     size_t getPrice() const override;
+
+    void nextDay() override;
+
     bool operator==(const Cargo& cargoToCheck) const override;
     Cargo& operator+=(size_t amount) override;
     Cargo& operator-=(size_t amount) override;
 
+
```
Comment (arafalski) [inline shm/inc/item.hpp]: Delete this empty line

File: shm/src/fruit.cpp [hunk 2]
```diff
@@ -19,6 +19,11 @@ size_t Fruit::getPrice() const {
     return static_cast<size_t>((static_cast<float>(leftTime_) / static_cast<float>(timeToSpoil_)) * static_cast<float>(basePrice_));
 }
 
+
```
Comment (arafalski) [inline shm/src/fruit.cpp]: Delete this line

File: shm/src/item.cpp [hunk 3]
```diff
@@ -9,6 +9,11 @@ size_t Item::getPrice() const {
     return basePrice_ * priceMultipliers_.at(rarity_);
 }
 
+
```
Comment (arafalski) [inline shm/src/item.cpp]: Delete

File: shm/src/player.cpp [hunk 4]
```diff
@@ -12,3 +12,10 @@ void Player::changeMoney(size_t amount) {
     }
     money_ += amount;
 }
+void Player::payCrew(size_t money) {
```
Comment (arafalski) [inline shm/src/player.cpp:16]: New line above this one

File: shm/inc/cargo.hpp [hunk 5]
```diff
@@ -15,6 +15,9 @@ class Cargo : public Cloneable<Cargo> {
     virtual size_t getPrice() const = 0;
     virtual size_t getBasePrice() const = 0;
 
+
```
Comment (arafalski) [inline shm/inc/cargo.hpp]: Delete

**Review by arafalski — APPROVED** [review]

**Review by kontrybutor — APPROVED** [review]

**Pull request merged**